projects/packages/videopress/src/class-site.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Provides site data sourced from WPCOM
+ *
+ * @package automattic/jetpack-videopress
+ */
+
+namespace Automattic\Jetpack\VideoPress;
+
+use Automattic\Jetpack\Connection\Client;
+use WP_Error;
+
+/**
+ * Provides site data sourced from WPCOM
+ */
+class Site {
+
+	/**
+	 * Returns all the data provided by WPCOM for the site.
+	 *
+	 * @return int|WP_Error the total of plays for today, or WP_Error on failure.
+	 */
+	public static function get_site_info() {
+		$error = new WP_Error(
+			'videopress_site_error',
+			__( 'Could not fetch site information from the service', 'jetpack-videopress-pkg' )
+		);
+
+		$blog_id      = VideoPressToken::blog_id();
```

Comment:
What do you think about adding a `get_blog_id()` method to the Data class?

Reply:
Nice, as a central point for getting this info.